charts/oceanbase-dashboard/templates/prom-rule-config.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
     name: {{ .Release.Name }}-prometheus-rules-conf
 data:
   prometheus.rules: |-
-    groups:
+    {{`groups:
```

Comment:
What is the usage of this pair of `{{` and `}}`

Reply:
to escape all the template marker {{ xx }}  in the config